masterlist.yaml

```
@@ -28190,3 +28190,49 @@ plugins:
         crc: 0x9E8F3CA5
         util: '[SSEEdit v4.0.4](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 17
+        
+  - name: 'UnforgivingDevices - DD Patch.esp'
+    url:
+      - link: 'https://www.loverslab.com/files/file/16938-unforgiving-devices/'
+        name: 'unforgiving Devices DD Patch'
+    after:
+      - 'Devious Devices SE patch.esp'
+      - 'Devious Devices - Assets.esm'
+      - 'Devious Devices - Expansion.esm'
+      - 'Devious Devices - Integration.esm'
+      - 'Devious Devices For Him.esp'
+      - 'Devious Devices - BRRF.esp'
+      - 'Devious Devices - Lore Patch.esp'
+    clean:
+      - crc: 0xD32A2D77
+        util: 'SSEEdit v4.0.4'
+  - name: 'UnforgivingDevices - LBS Patch.esp'
+    url:
+      - link: 'https://www.loverslab.com/files/file/16938-unforgiving-devices/'
+        name: 'unforgiving Devices Lauras Bondage Shop Patch'
+    after:
+      - 'UnforgivingDevices - DD Patch.esp'
+      - 'Laura's Bondage Shop.esp'
+    clean:
+      - crc: 0x5EDE9478
+        util: 'SSEEdit v4.0.4'
+  - name: 'UnforgivingDevices - Pink Patch.esp'
+    url:
+      - link: 'https://www.loverslab.com/files/file/16938-unforgiving-devices/'
+        name: 'Unforgiving Devices Devious Pink Patch'
+    after:
+      - 'UnforgivingDevices - DD Patch.esp'
+      - 'DeviouslyPink.esp'
+    clean:
+      - crc: 0x907DCAE7
+        util: 'SSEEdit v4.0.4'
+  - name: 'UnforgivingDevices - Security Overhaul Patch.esp'
+    url:
+      - link: 'https://www.loverslab.com/files/file/16938-unforgiving-devices/'
+        name: 'unforgiving Devices Devious Cursed Loot Patch'
+    after:
+      - 'UnforgivingDevices - DD Patch.esp'
+      - 'Deviously Cursed Loot.esp'
+    clean:
+      - crc: 0xB694000E
+        util: 'SSEEdit v4.0.4'
```

There should be an extra newline at the end of the file:

```suggestion
        util: 'SSEEdit v4.0.4'

```